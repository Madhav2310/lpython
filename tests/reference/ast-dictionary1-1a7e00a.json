{
    "basename": "ast-dictionary1-1a7e00a",
    "cmd": "lpython --show-ast --no-color {infile} -o {outfile}",
    "infile": "tests/dictionary1.py",
    "infile_hash": "07b1b2b7340dc55f22cdb56c0868cac9d0de39c1e93da9af95885557",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "ast-dictionary1-1a7e00a.stdout",
    "stdout_hash": "ae11b6ab5379d49dcfa8d7e82fbfec6beae05f794d5b6cdc85c8ed0e",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}