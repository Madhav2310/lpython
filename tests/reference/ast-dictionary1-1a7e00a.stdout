(Module [(FunctionDef test_Dict ([] [] [] [] [] [] []) [(AnnAssign (Name x Store) (Subscript (Name dict Load) (Tuple [(Name i32 Load) (Name i32 Load)] Load) Load) () 1) (Assign [(Name x Store)] (Dict [(ConstantInt 1 ()) (ConstantInt 3 ())] [(ConstantInt 2 ()) (ConstantInt 4 ())]) ()) (AnnAssign (Name y Store) (Subscript (Name dict Load) (Tuple [(Name str Load) (Name i32 Load)] Load) Load) () 1) (Assign [(Name y Store)] (Dict [(ConstantStr "a" ()) (ConstantStr "b" ())] [(UnaryOp USub (ConstantInt 1 ())) (UnaryOp USub (ConstantInt 2 ()))]) ()) (AnnAssign (Name z Store) (Name i32 Load) () 1) (Assign [(Name z Store)] (Subscript (Name y Load) (ConstantStr "a" ()) Load) ()) (Assign [(Name z Store)] (Subscript (Name y Load) (ConstantStr "b" ()) Load) ()) (Assign [(Name z Store)] (Subscript (Name x Load) (ConstantInt 1 ()) Load) ())] [] () ())] [])
